universal-application-tool-0.0.1/app/views/components/QuestionBank.java

```diff
@@ -176,6 +174,11 @@ private ContainerTag renderQuestionDefinition(QuestionDefinition definition) {
     return questions.stream().filter(filter).collect(ImmutableList.toImmutableList());
   }
 
+  /** If a block already contains a single-block question, no more questions can be added. */
+  private boolean noMoreQuestionsAllowed() {
```

Comment:
(blocking) rename `noMoreQuestionsAllowed` to `containsSingleBlockQuestion` to keep re-using the new "single block question" term.

Reply:
Done.